app/views/devise/passwords/new.html.slim

```diff
@@ -1,14 +1,9 @@
 - title t('titles.passwords.forgot')
 
-- if decorated_session.sp_name == 'CBP Trusted Traveler Programs'
-  .alert.alert-notice
-    .bold = 'Coming from Trusted Traveler Programs?'
-    p
-      = 'Your old GOES userID and password won’t work. Please '
-      = link_to 'create a login.gov account.', sign_up_email_url(request_id: params[:request_id])
-    p.mb1
-      = link_to 'Learn more.',
-                'https://login.gov/help/trusted-traveler-programs/sign-in-doesnt-work/'
+// - if decorated_session.sp_name == 'CBP Trusted Traveler Programs'
```

Comment:
We don't want to make the check anymore?

Reply:
@tbaxter-18f we do, I just came back to this branch and pushed it up so I could have a view of the changes. 

We have multiple SP specific messages so I'm working on drying it up.